gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/query/HiveAvroORCQueryGenerator.java

```diff
@@ -202,7 +204,21 @@ public static String generateCreateTableDDL(Schema schema,
     //    .. use columns from destination schema
     if (isEvolutionEnabled || !destinationTableMeta.isPresent()) {
       log.info("Generating DDL using source schema");
+      System.out.println("Generating DDL using source schema");
       ddl.append(generateAvroToHiveColumnMapping(schema, Optional.of(hiveColumns), true, dbName + "." + tblName));
+      try {
```

Comment:
I want to make sure only when schema evolution is enabled or there is no existing table, we can set the columns. Because that will overwrite the schema.

Reply:
Fair enough. Worth to check if the production have schema evolution enabled.

Reply:
Yes, at least it's enabled in scoreevent. BTW, can you look at the new pr for this commit? I have sent you the email. Thank you!